### PR TITLE
{perf}[system/system] Austin v3.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/a/Austin/Austin-3.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+homepage = 'https://github.com/P403n1x87/austin'
+description = "Austin is a Python frame stack sampler for CPython written in pure C."
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/P403n1x87/austin/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['ee189af905fac77ff9173b57d8e927bdb7c4cf5e18b1bfd7f4456ac46fe04484']
+
+cmds_map = [('.*', "$CC -std=c99 -O3 -Os -Wall -pthread src/*.c -o src/austin")]
```
Comment (lexming) [inline easybuild/easyconfigs/a/Austin/Austin-3.2.0.eb:15]: Is it necessary to compile with `-Os`? I would use `$CFLAGS` or `$OPTFLAGS` here.
```suggestion
cmds_map = [('.*', "$CC $CFLAGS -std=c99 -Wall -pthread src/*.c -o src/austin")]
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2651.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/2aa7a3c2b4d6eee7d987819a763440ea for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3539.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/9e07c65700cebaa0e18c1871a323855f for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2dede6673d4e5ceff9cfd35c5d3352d9 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**